src/schema/artwork/index.ts

```diff
@@ -577,6 +577,9 @@ export const artworkFields = () => {
       description:
         "The string that describes domestic and international shipping.",
       resolve: artwork => {
+        if (!artwork.acquireable)
+          return null
+
```

Comment:
So to follow last comment on this one from Steven: https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=57&projectKey=DISCO&modal=detail&selectedIssue=DISCO-494 - I think this case and the one that is right after should be combined and just do something like:
```if (!artwork.acquireable || artwork.domestic_shipping_fee_cents == null &&
          artwork.international_shipping_fee_cents == null)
return null
```

I think that will effectively remove this shipping lone for all works except acquirable ones that have shipping set to something.

Reply:
:+1: